.github/workflows/ci_workflows.yml

```
@@ -0,0 +1,82 @@
+name: CI
+
+on:
+  push:
+  #  branches:  #uncomment to run only on master
+  #  - master
+  #  tags: # run CI if specific tags are pushed
+  pull_request:
+  #  branches: # only build on PRs against 'main' if you need to further limit when CI is run.
+  #  - main
```

Since you are still using `master`, not `main`.
```suggestion
  #  - master
```